DFC.ServiceTaxonomy.ApiFunction.Tests/ExecuteHttpTriggerTests.cs

```diff
@@ -0,0 +1,232 @@
+            Assert.IsAssignableFrom<IActionResult>(result);
+            Assert.True(result is InternalServerErrorResult);
+            Assert.Equal((int?)HttpStatusCode.InternalServerError, internalServerErrorResult.StatusCode);
+        }
+        
+        [Fact]
+        public async Task Execute_WhenRequestBodyDoesntContainFieldsForCypherQuery_ReturnsBadRequestErrorMessageResult()
+        {
+            _config.Value.Function = "GetAllSkills";
+            var query = "{\"query\": \"QUERY HERE\", \"queryParam\": [{\"name\": \"occupation\"}]}";
+            A.CallTo(() => _httpRequestHelper.GetBodyFromHttpRequest(_request)).Returns(@"{ }");
+
+            A.CallTo(() => _fileHelper.ReadAllTextFromFile("CypherQueries\\GetAllSkills.json")).Returns(query);
+
+            A.CallTo(() => _jsonHelper.DeserializeObject<Cypher>(query)).Returns(_cypherModel);
+
+            var result = await RunFunction();
+
+            var badRequestObjectResult = result as BadRequestObjectResult;
+
+            // Assert
+            Assert.IsAssignableFrom<IActionResult>(result);
+            Assert.True(result is BadRequestObjectResult);
+            Assert.Equal((int?)HttpStatusCode.BadRequest, badRequestObjectResult.StatusCode);
+        }
+        
+        [Fact]
+        public async Task Execute_WhenCodeIsValid_ReturnsOkObjectResult()
+        {
+            _config.Value.Function = "GetAllSkills";
+            var query = "{\"query\": \"QUERY HERE\", \"queryParam\": [{\"name\": \"occupation\"}]}";
+            A.CallTo(() => _httpRequestHelper.GetBodyFromHttpRequest(_request)).Returns(@"{ ""occupation"": ""http://data.europa.eu/esco/occupation/5793c124-c037-47b2-85b6-dd4a705968dc"" }");
+
+            A.CallTo(() => _fileHelper.ReadAllTextFromFile("CypherQueries\\GetAllSkills.json")).Returns(query);
+
+            A.CallTo(() => _jsonHelper.DeserializeObject<Cypher>(query)).Returns(_cypherModel);
+
+            var dict = new Dictionary<string, object>
+            {
+                {"occupation", "http://data.europa.eu/esco/occupation/5793c124-c037-47b2-85b6-dd4a705968dc"}
+            };
+
+            var statementResult = A.Fake<IStatementResultCursor>();
+            var resultSummary = A.Fake<IResultSummary>();
+            var records = new List<object> {new object()};
+
+            A.CallTo(() => _neo4JHelper.ExecuteCypherQueryInNeo4JAsync("query", dict)).Returns(statementResult);
+            A.CallTo(() => _neo4JHelper.GetListOfRecordsAsync(statementResult)).Returns(records);
+            A.CallTo(() => _neo4JHelper.GetResultSummaryAsync(statementResult)).Returns(resultSummary);
+            
+            var result = await RunFunction();
+
+            var okObjectResult = result as OkObjectResult;
+
+            // Assert
+            Assert.IsAssignableFrom<IActionResult>(result);
+            Assert.True(result is OkObjectResult);
+            Assert.Equal((int?)HttpStatusCode.OK, okObjectResult.StatusCode);
+        }
+
```

Comment:
It'd be good if we had a happy path test, that fakes a json response from neo, and checks that we pick out and return the correct subset of the json response.